tools/launch.ts

```diff
@@ -352,7 +352,7 @@ async function start() {
       }
       // If it is an array, push the position at which we are
       else if (Array.isArray(argv["parachain-chain"])) {
-        parachainsChains.push(argv["parachain-chain"] || parachains[parachainName].chain);
+        parachainsChains.push(argv["parachain-chain"][i] || parachains[parachainName].chain);
```

Comment:
Are you sure about this ?

Reply:
the assumption here is that you will have the same number of --parachain-chain arguments as --parachain arguments. I can try to re-work this to reflect better the case in which this is not true